j5test/test_Utils.py

```diff
@@ -11,39 +20,39 @@ class SubvalueError(ValueError):
 
 def sample_method(option):
     if option == 1:
-        return "pigeons"
+        return text_to_native_str("pigeons")
```

Comment:
What is the reason that these need to be native strings?

Reply:
If they are unicode strings then they cause some tests to fail because they add a `u` prefix. This seemed like the easiest solution to fix the tests.

Reply:
ok